translations/README/GERMAN.md

```diff
@@ -79,22 +79,22 @@ Du wirst eine Kopie von dem Karten-Template innerhalb der HTML Datei machen und
 
 Diese Anleitung ist auch in anderen Sprachen verfügbar:  [other languages](/translations/README.md)
 
-|     [Arabic (عربي)](/translations/README/ARABIC.md)     |  [Bangla (বাংলা)](/translations/README/BANGLA.md)  | [Chinese (Traditional) (繁體中文)](/translations/README/CHINESE_TRADITIONAL.md) |            [English (English)](/README.md)             |  [French (Français)](/translations/README/FRENCH.md)  
+|     [Arabisch (عربي)](/translations/README/ARABIC.md)     |  [Bangla (বাংলা)](/translations/README/BANGLA.md)  | [Chinesisch (Traditionell) (繁體中文)](/translations/README/CHINESE_TRADITIONAL.md) |            [Englisch (English)](/README.md)             |  [Französisch (Français)](/translations/README/FRENCH.md)  
 | :---------------------------------------------: | :---------------------------------------: | :-----------------------------------------: | :---------------------------------------: | :---------------------------------------: |
-|  [German (Deutsch)](/translations/README/GERMAN.md)  |      [Hindi (हिंदी)](/translations/README/HINDI.md)      | [Italian (Italiano)](/translations/README/ITALIAN.md) | [Japanese (日本語)](/translations/README/JAPANESE.md) |  [Korean (한국어)](/translations/README/KOREAN.md)  |
-  [Polish (Polski)](/translations/README/POLISH.md)  | [Portuguese (Portuguese)](/translations/README/PORTUGUESE.md) | [Russian (Русский)](/translations/README/RUSSIAN.md) |  [Serbian (Српски)](/translations/README/SERBIAN.md)  | [Spanish (Español)](/translations/README/SPANISH.md) |
-   [Turkish (Türkçe)](/translations/README/TURKISH.md) | [Ukrainian (українська)](/translations/README/UKRAINIAN.md) | [Norwegian (Norsk)](/translations/README/NORWEGIAN.md)
+|  [Deutsch (Deutsch)](/translations/README/GERMAN.md)  |      [Hindi (हिंदी)](/translations/README/HINDI.md)      | [Italienisch (Italiano)](/translations/README/ITALIAN.md) | [Japanisch (日本語)](/translations/README/JAPANESE.md) |  [Koreanisch (한국어)](/translations/README/KOREAN.md)  |
+  [Polnisch (Polski)](/translations/README/POLISH.md)  | [Portugisisch (Portuguese)](/translations/README/PORTUGUESE.md) | [Russisch (Русский)](/translations/README/RUSSIAN.md) |  [Serbisch (Српски)](/translations/README/SERBIAN.md)  | [Spanisch (Español)](/translations/README/SPANISH.md) |
+   [Turkisch (Türkçe)](/translations/README/TURKISH.md) | [Ukrainisch (українська)](/translations/README/UKRAINIAN.md) | [Norwegisch (Norsk)](/translations/README/NORWEGIAN.md)
 
-> Übersetzungen für Projekt-Dokumentationen sind stets willkommen. Lies [`Translation Guide`](/translations/README.md) um beizutragen.
+> Übersetzungen für Projekt-Dokumentationen sind stets willkommen. Lies den [`Translation Guide`](/translations/README.md) um beizutragen.
 
 ---
 
 ### Installation :)
 
-> [!Note]
+> [!Notiz]
```

Comment:
```suggestion
> [!Note]
```
it's a GitHub markdown syntax, changing it will break the markdown and will serve no purpose. The same goes for relative links, external links, and guides.

Reply:
I think I resolved the problem and changed it back to "Note". Sorry, didn't know it was a markdown syntax thing., still very new to working with GitHub and markdown :)